2022/09.md

```diff
@@ -95,7 +95,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 2     | 60m     | [Array.fromAsync](https://github.com/tc39/proposal-array-from-async) for Stage 3 (slides TBD) | J. S. Choi |
     |   | 2     | 60m     | Set methods discussion ([slides](https://docs.google.com/presentation/d/1HCqPMsWiTtsn92gA3b1luVpnVHWVVR0iKaAE0marxkA)) | Kevin Gibbons |
     |   | 2     | 60m     | [Explicit Resource Management](https://github.com/tc39/proposal-explicit-resource-management) for Stage 3 ([slides](https://1drv.ms/p/s!AjgWTO11Fk-TkoEvgEld8y2bGCjTlw?e=ySkk5z), [spec](https://tc39.es/proposal-explicit-resource-management/)) | Ron Buckton |
-    |   | 1     | 15m     | [Iteration helpers as functions](https://github.com/rauschma/iterable) for stage 1 (slides TBD) | Axel Rauschmayer |
+    |   | –     | 15m     | Discussion: switching [iterator helpers](https://github.com/rauschma/iterable) to functions ([slides](https://speakerdeck.com/rauschma/iteration-helper-functions)) | Axel Rauschmayer |
```

Comment:
I would avoid linking the term "iterator helpers" to anything other than the existing iterator helpers proposal repo, to avoid confusion.

```suggestion
    |   | 0     | 15m     | Discussion: switching [iterator helpers](https://github.com/tc39/proposal-iterator-helpers) to [functions](https://github.com/rauschma/iterable) ([slides](https://speakerdeck.com/rauschma/iteration-helper-functions)) | Axel Rauschmayer |
```

Reply:
Yes, my bad, I got the link wrong. Thanks!